modules/luci-mod-network/htdocs/luci-static/resources/view/network/routes.js

```diff
@@ -183,6 +183,11 @@ return view.extend({
 			o.datatype = 'uinteger';
 			o.placeholder = 10;
 
+			o = s.taboption('advanced', form.Value, 'uidrange', _('User identifier (UID)'));
+			o.modalonly = true;
+			o.datatype = 'string';
+			o.placeholder = '1000/1000-1005';
```

Comment:
The placeholder mangles two alternative values together, right?  (a single value and a range)
Might be more clear just to put here one legitimate value, instead of an illegal value.
Better to have the allowed option values explained in the help text.